roles/addons/ofed/tasks/main.yml

```diff
@@ -13,6 +13,18 @@
   tags:
     - package
 
+- name: configure system memlock settingsa for rdma
+  pam_limits:
+    domain: '*'
+    limit_type: "{{item.limit_type}}"
+    limit_item: "{{item.limit_item}}"
+    value: "{{item.value}}"
+  with_items:
+    - { limit_type: 'soft', limit_item: 'memlock', value: unlimited }
+    - { limit_type: 'hard', limit_item: 'memlock', value: unlimited }
```

Comment:
I would prefer to create tunable here:
```
- { limit_type: 'soft', limit_item: 'memlock', value: {{ ofed_memlock_soft }} }
- { limit_type: 'hard', limit_item: 'memlock', value: {{ ofed_memlock_hard }} }
```

Then define both variables in default/main.yml with value unlimited if it is Mellanox recommendation:
```
ofed_memlock_soft: 'unlimited'
ofed_memlock_hard: 'unlimited'
```

Reply:
Sorry Bruno! I'm new with BlueBanquise, new with Ansible and new with Github too :-)
I hope this version is better than the first one.

Reply:
No problem, this is the purpose of code reviews. :smiley: